pkg/controller/job/utils.go

```diff
@@ -29,3 +29,13 @@ func IsJobFinished(j *batch.Job) bool {
 	}
 	return false
 }
+
+func FilterActive(jobs *batch.JobList) (active []*batch.Job) {
```

Comment:
Prefer to put this function into the e2e test, and not turn the job controller package into a library.